PWGJE/Tasks/trackEfficiency.cxx

```diff
@@ -76,18 +76,28 @@ struct TrackEfficiencyJets {
   Configurable<std::vector<double>> centralityBinning{"centralityBinning", {0., 10., 50., 70.}, "binning of centrality histograms"};
   Configurable<int> intRateNBins{"intRateNBins", 50, "number of bins for interaction rate axis"};
   Configurable<float> intRateMax{"intRateMax", 50000.0, "maximum value of interaction rate axis"};
+  Configurable<int> phiEffNBins{"phiEffNBins", 200, "number of bins for phi axis in efficiency plots"};
+  Configurable<int> etaEffNBins{"etaEffNBins", 200, "number of bins for eta axis in efficiency plots"};
+
+  Configurable<float> ptHatMin{"ptHatMin", 5, "min pT hat of collisions"};
+  Configurable<float> ptHatMax{"ptHatMax", 300, "max pT hat of collisions"};
+  Configurable<float> pTHatExponent{"pTHatExponent", 6.0, "exponent of the event weight for the calculation of pTHat"};
 
   std::vector<int> eventSelectionBits;
   int trackSelection = -1;
+  int acceptSplitCollisionsCase0 = 0; // acceptSplitCollisions case: only look at mcCollisions that are not split
+  int acceptSplitCollisionsCase2 = 2; // acceptSplitCollisions case: accept split mcCollisions but only look at the first reco collision associated with it
```

Comment:
Where are these values defined?

Reply:
This implementation is not easy to read, understand, use and maintain, because the names do not reflect the meanings of the values and there is not sanity check to validate the input values provided in `acceptSplitCollisions`.
I recommend to:
- replace these with `enum` with meaningful names,
- add a sanity check for `acceptSplitCollisions` in `init` which throws a `fatal` in case of invalid values.
- branch the code with `if`/`else if` blocks without an `else` block.

Reply:
Hi @vkucera 
I was away until this morning, I just updated my file to address your comments, I hope everything looks better now.